spec/lib/fields/time_spec.rb

```diff
@@ -13,4 +13,30 @@
       expect(path).to eq("/fields/time/#{page}")
     end
   end
+
+  describe "#time" do
+    let(:time) { DateTime.new(2021, 3, 26, 16, 38) }
+    let(:formats) do
+      {
+        time: {
+          formats: { short: "%H:%M" },
+        },
+      }
+    end
```

Comment:
A minor point: we typically avoid using `let` in this codebase and prefer to either inline a variable in each `it` or extract it into a normal Ruby method.